### PR TITLE
[TR] [PAM] PIM-4420: Add events in pim-dialogform.js when it opens

### PR DESCRIPTION
**Pull request by willy-ahva**
| Q                    | A
| -------------------- | ---
| Bug fix?             | n
| New feature?         | n
| BC breaks?           | n
| CI currently passes? | ?
| Tests pass?          | y
| Scenarios pass?      | -
| Checkstyle issues?*  | n
| PMD issues?**        | n
| Changelog updated?   | n
| Fixed tickets        | PIM-4420
| DB schema updated?   | n
| Migration script?    |
| Doc PR               |

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Resources/public/js/pim-dialogform.js [hunk 1]
```diff
@@ -131,6 +134,7 @@ define(
                     success: function (data) {
                         loadingMask.hide();
                         createDialog(data);
+                        mediator.trigger('dialog:open:after', this);
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/js/pim-dialogform.js]: dialog:open:after means: AFTER the OPENning of a DIALOG and here, this is the closing of the modal. You should not name this event like that or remove it

Comment (willy-ahva) [inline src/Pim/Bundle/UIBundle/Resources/public/js/pim-dialogform.js]: Here you open the dialog form. Here you close it https://github.com/akeneo/pim-community-dev/blob/PIM-4420/src/Pim/Bundle/UIBundle/Resources/public/js/pim-dialogform.js#L73 .

**Comment by grena** [comment]
GTM

**Comment by sumbobyboys** [comment]
GTM

**Pull request merged**